packages.md

```diff
@@ -8,7 +8,7 @@
 | [`Azure.Data.AppConfiguration`]( sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj )| [Readme](sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) | [Changelog](sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md) | [![Azure.Data.AppConfiguration](https://img.shields.io/nuget/vpre/Azure.Data.AppConfiguration.svg)]( https://www.nuget.org/packages/Azure.Data.AppConfiguration/ ) |
 | [`Azure.Identity`]( sdk/identity/Azure.Identity/src/Azure.Identity.csproj )| [Readme](sdk/identity/Azure.Identity/README.md) | [Changelog](sdk/identity/Azure.Identity/CHANGELOG.md) | [![Azure.Identity](https://img.shields.io/nuget/vpre/Azure.Identity.svg)]( https://www.nuget.org/packages/Azure.Identity/ ) |
 | [`Azure.Messaging.EventHubs`]( sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj )| [Readme](sdk/eventhub/Azure.Messaging.EventHubs/README.md) | [Changelog](sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md) | [![Azure.Messaging.EventHubs](https://img.shields.io/nuget/vpre/Azure.Messaging.EventHubs.svg)]( https://www.nuget.org/packages/Azure.Messaging.EventHubs/ ) |
-| [`Azure.Messaging.EventHubs.CheckpointStore.Blobs`]( sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/src/Azure.Messaging.EventHubs.CheckpointStore.Blobs.csproj )| [Readme](sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/README.md) | [Changelog](sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/CHANGELOG.md) | [![Azure.Messaging.EventHubs.CheckpointStore.Blobs](https://img.shields.io/nuget/vpre/Azure.Messaging.EventHubs.CheckpointStore.Blobs.svg)]( https://www.nuget.org/packages/Azure.Messaging.EventHubs.CheckpointStore.Blobs/ ) |
+| [`Azure.Messaging.EventHubs.Processor`]( sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Azure.Messaging.EventHubs.Processor.csproj )| [Readme](sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md) | [Changelog](sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md) |  N/A  |
 | [`Azure.Security.KeyVault.Certificates`]( sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj )| [Readme](sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md) | [Changelog](sdk/keyvault/Azure.Security.KeyVault.Certificates/ChangeLog.md) | [![Azure.Security.KeyVault.Certificates](https://img.shields.io/nuget/vpre/Azure.Security.KeyVault.Certificates.svg)]( https://www.nuget.org/packages/Azure.Security.KeyVault.Certificates/ ) |
 | [`Azure.Security.KeyVault.Keys`]( sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj )| [Readme](sdk/keyvault/Azure.Security.KeyVault.Keys/README.md) | [Changelog](sdk/keyvault/Azure.Security.KeyVault.Keys/ChangeLog.md) | [![Azure.Security.KeyVault.Keys](https://img.shields.io/nuget/vpre/Azure.Security.KeyVault.Keys.svg)]( https://www.nuget.org/packages/Azure.Security.KeyVault.Keys/ ) |
```

Comment:
The PR description said all changelogs were upper-cased, but that isn't the case here, I see (and for other Key Vault changelogs).

Reply:
Right, the actual files have been renamed. Looks like I just need to re-index.